nautilus_open_any_terminal/open_any_terminal_extension.py

```diff
@@ -112,6 +112,36 @@
     "tabby": "-e",
 }
 
+TERM_NAME = {
+    "alacritty": "Alacritty",
+    "blackbox": "BlackBox",
+    "cool-retro-term": "cool-retro-term",
+    "deepin-terminal": "Deepin Terminal",
+    "foot": "Foot",
+    "footclient": "FOOTCLIENT",
```

Comment:
why is this all capital? Not sure this is wanted.

Reply:
Is FootClient good?

Reply:
I would prefer it yes. Having this all bold looks weird to me

Reply:
done 👍